app/helpers/team_helper.rb

```diff
@@ -1,19 +1,19 @@
 module TeamHelper
-  WHITELISTED_DOMAINS = %w[.mil .gov .fed.us].freeze
+  ALLOWED_DOMAINS = /@.+\.(mil|gov|fed\.us|state\..+\.us)$/
```

Comment:
I think this would be a good spot to use the `mail` gem to extract the domain and only check that part, rather than keep that as part of the regex, [people can get sneaky and add multiple @'s inside an address](https://stackoverflow.com/a/12355882)

```ruby
Mail::Address.new('"foobar@bar.baa"@bar.com').domain
# => "bar.com"
```

Reply:
I totally agree. I'm in a tricky spot because we have a partner that has a `state.*.us` email address and it's limiting their ability to create teams and use the Dashboard as intended. This is sort of a patch. 

Do you think this would be too risky as-is? This still relies on Login for the actual login process so would Login let a user have an email address formatted like that?

Reply:
Just to be clear!! I was saying let's drop regexing for `@` and just extract the domain and regex that part

I tried for 5 seconds to break the current regex and failed, I think the anchor `$` saves us from anything egregious, so I wouldn't worry about this for now

Reply:
<3 I think we can also add the team via the console for now and then I can give myself a little more time to work on this and explore using the gem.